### ISSUE TITLE
[Bug]: Blade not recognizing variables using View::make()->with()

### ISSUE THREAD

**Issue by peterpan666** [laravel-idea/plugin#643]
### Bug description

Hi,

When I use `View::make()->with()` the blade view can't recognize the variables inside the `with` method.

### Plugin version

5.8.2.222

### Operating system

MacOS

### Steps to reproduce

Inside a controller, use the following :
```php
return \View::make('home')->with('foo', collect());
```
Inside the blade view, you can't use the autocompletion on `$foo`.

Now with :
```php
return view('home')->with('foo', collect());
```

Everything is working perfectly which doesn't make sense to me since both solutions have exactly the same signature.

Is there anything to do about this ?

Thank you

**Comment by adelf** [comment]
Hi, Elie.

Thanks for reporting. Will be fixed.

**Comment by peterpan666** [comment]
Awesome thanks !

**Comment by adelf** [comment]
Fixed in the 6.2.5 version. Thank you.

**Comment by peterpan666** [comment]
Tested, seems to be working, thanks !
